public/mix-manifest.json

```diff
@@ -1,10 +1,6 @@
 {
-    "/js/app.js": "/js/app.js?id=65c22c70a408625c7e72",
-    "/css/app.css": "/css/app.css?id=2984556e5f69d69788e7",
-    "/css/stripe.css": "/css/stripe.css?id=956554a2e96db30fafa3",
-    "/js/app.js.map": "/js/app.js.map?id=e62443022dbabf2c461f",
-    "/css/app.css.map": "/css/app.css.map?id=a1585d15bdf5306642f7",
-    "/css/stripe.css.map": "/css/stripe.css.map?id=80d7af9d49b0922d35e6",
-    "/js/stripe.js": "/js/stripe.js?id=55d823b61fc4f7e8b1ab",
-    "/js/stripe.js.map": "/js/stripe.js.map?id=4f29019f2c2bda216ee1"
+    "/js/app.js": "/js/app.js?id=f6844194a630b44f9e48",
+    "/css/app.css": "/css/app.css?id=2b58773eb3b4f2c35795",
+    "/css/stripe.css": "/css/stripe.css?id=c16ddc73ba6af94a5bfa",
+    "/js/stripe.js": "/js/stripe.js?id=b3b3f79ea16865ee5107"
```

Comment:
You want to compile assets with `npm run production`, not in `dev` mode.

Reply:
Done